src/components/shared/ProfileMetainfoSummary.css

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* We use em in this CSS file so that it will adjust with the inherited font-size. */
+
+.profileMetainfoSummary {
```

Comment:
Super nit, optional: I would think camel case would be `ProfileMetaInfoSummary`.

Reply:
I think we have this discrepency since day 1, I'll look at fixing this everywhere :)